mathjax3-ts/handlers/html/HTMLDocument.ts

```diff
@@ -228,7 +228,7 @@ export class HTMLDocument extends AbstractMathDocument {
     public TestMath(text: string, display: boolean = true) {
         if (!this.processed['TestMath']) {
             let math = new HTMLMathItem(text, this.inputJax[0], display);
-            math.setMetrics(6, 14, 1000000, 1000000, 1);
+            math.setMetrics(16, 8, 1000000, 1000000, 1);
```

Comment:
Do we want to name these magic numbers? E.g., defaultEm, default,Ex

Reply:
I'm not sure I understand how.  Do you mean by passing an object instead?  They are, of course, named in the definition of `setMetrics()` in [`core/MathItem.ts`](https://github.com/mathjax/mathjax-v3/blob/master/mathjax3-ts/core/MathItem.ts#L172).  Do you have something else in mind?

(Also, this particular function is temporary, and is only here for testing purposes, so these particular values are not going to be in the final version, since this function will be removed.)